README.md

```diff
@@ -197,13 +197,15 @@ custom:
           ExtendedStatistic: p99
           Threshold: 5000
       States: # Step Functions
-        Statistic: Sum
         ExecutionsThrottled:
           Threshold: 0
+          Statistic: Sum
```

Comment:
From what I can tell, `Statistic: Sum` declared at the `States` level should cascade to the individual metrics, so there shouldn't be a need to explicitly copy it for each one. Did I miss something? Is there a reason for this?

Reply:
Oh I didn't realize it would cascade. I thought it was a typo in the readme as it seemed inconsistent with how the other alarms were specified. I can revert it!